kubos/main.py

```diff
@@ -70,6 +71,8 @@ def main():
     local_config = sdk_config.load_config()
     add_kubos_command = functools.partial(kubos_options.command.add_command, local_config, subparser, 'kubos') #add our own implemented commands
     add_yotta_command = functools.partial(kubos_options.command.add_command, local_config, subparser, 'yotta') #add from the default yotta commands
+    add_kubos_command_sync = functools.partial(kubos_options.command.add_command_sync, local_config, subparser, 'kubos') #add our own implemented commands
```

Comment:
nit: can you move this to be immediately before or after the other kubos command line? 
Also, please tweak the comments for the these two add_kubos_command* lines to mention that one is for async commands and one is for sync, rather than just having the same comment line for both.